src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,14 +1,42 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
+    @NotNull
+    @Size(min = 8)
     private String password;
+    @NotNull
+    @Size(min = 8)
+    private String repeatPassword;
```

Comment:
checkstyle: Don’t repeat checks for fields: you must ensure email field not to be null in your custom Validator, so there is no need to put @NotNull for that field as well in Dto class